centreon/www/include/configuration/configObject/meta_service/DB-Func.php

```
@@ -1,1182 +1,572 @@
-                if (! testExistence($metaName)) {
-                    continue;
-                }
-                $params = [];
-                foreach ($row as $column => $value) {
-                    $params[] = QueryParameter::string($column, $value);
-                }
-                $pearDB->insert($insertQuery, QueryParameters::create($params));
-                $newMetaId = $pearDB->getLastInsertId();
-                if ($newMetaId) {
+    # Foreach Meta Service
+    foreach ($metas as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)